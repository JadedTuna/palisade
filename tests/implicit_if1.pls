in {
    high x: bool;
}
out {
    low y: int;
}

if (x) {
    y = 1;
} else {
    y = -1;
}
