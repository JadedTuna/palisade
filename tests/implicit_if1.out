[1;34mnote: [0mlabel of [1;34my[0m set to [1;93mhigh[0m
   9 |     y [1;34m=[0m 1;
       [1;34m~~~~~~[0m[1;34m^[0m
[1;34mnote: [0mlabel of [1;34my[0m set to [1;93mhigh[0m
  11 |     y [1;34m=[0m -1;
       [1;34m~~~~~~[0m[1;34m^[0m
[[1;31mFAIL[0m] [1;93mlow [0m [1;34my[0m is [1;93mhigh[0m
